File: js/programmers/level_0/머쓱이보다 키 큰 사람.js
// Lv. 0 머쓱이보다 키 큰 사람

/**
 * @param {number[]} array
 * @param {number} height
 * @return {number}
 */

const solution = (array, height) => {
    return array.filter((v) => v > height).length;
};

File: java/programmers/level_0/src/머쓱이보다_키_큰_사람.java
public class 머쓱이보다_키_큰_사람 {

  public int solution(int[] array, int height) {
    int answer = 0;
    for (int i = 0; i < array.length; i++) {
      if (array[i] > height) {
        answer++;
      }
    }
    return answer;
  }
}
